releases/example.yaml

```diff
@@ -1,17 +1,19 @@
 ---
-version: 0.5.0
+version: v0.5.0
```

Comment:
Can we test with >v0.6.0 so we have all the subctl features?

Reply:
What do you mean? This is just an example, when we start using it it'll use whatever subctl you tell it to use

Reply:
Thanks mike, sorry for the issue.